Python_Backend/AnswerGenerator.py

```diff
@@ -12,6 +12,15 @@ class FaceState(Enum):
     SMILE = 1 
     FROWN = 2
 
+class IrisState(Enum):
+    EYES_UP = 0

```

Comment:
Can we make these Enuims strings? so that the value is just the meaningful value? or do you index by these?

Reply:
I see you're referring to them by index a lot so disregard.